packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -302,6 +302,100 @@ Use the `add` operation to add new claims to a token. If you use the `add` opera
   ]
 }
 ```
+
+You can also use the `add` operation to add new members to existing JSON objects and new elements to existing arrays. For example, you have a JSON object in a claim called `employee_profile`, and you want to add the `department_id` member to the claim.
```

Comment:
so we are using `member` word instead of `property`? I think it is better to use `property` actually sine it is used more in the context of JSON, e.g. https://www.w3schools.com/js/js_json_objects.asp.
@venkatviswanathan-okta ^

Reply:
got it, `member` is used within [JSON patch spec](https://tools.ietf.org/html/rfc6902), so my comment isn't relevant.